pkg/storage/interfaces.go

```diff
@@ -112,13 +112,17 @@ type Interface interface {
 	// resourceVersion may be used to specify what version to begin watching,
 	// which should be the current resourceVersion, and no longer rv+1
 	// (e.g. reconnecting without missing any updates).
+	// If resource version is "0", this interface will get current object at given key
+	// and send it in an "ADDED" event, before watch starts.
```

Comment:
From an API client's perspective, the watch cache has already changed that behavior